### ISSUE TITLE
[tx] Handle fulltext in transactor

### ISSUE THREAD

**Issue by ncalexan** [mozilla/mentat#189]
There's enough work here to track separately.  It's identical to the Clojure implementation, but requires special care and testing.

It might be possible to implement #12 / #69 concurrently to this, since fulltext values are the first natural candidate for such a hashing scheme.

**Comment by ncalexan** [comment]
I'm trying to roadmap the path to a complete transactor, and I'd like to discuss whether fulltext indexing is a priority for us.

For the initial Clojure{Script} implementation, fulltext indexing _was_ a priority, for two reasons:
- our principle consumer was [Tofino](https://github.com/mozilla/tofino), which we expected to experiment with indexing whole web pages;
- and we were also looking to try to make it easier to fulltext index in Firefox add-ons by improving upon the SQLite story.

The consumer side of this story has changed entirely; Tofino is no longer driving any technology choices.  And depending on the targeted Firefox add-on or feature, it might be that fulltext indexing is a priority (for Activity Stream, yes; for request auto complete or certificate stores, no).

@joewalker, do you have a bead on the priority of fulltext indexing?  After #283 it won't be a huge effort to support, but it's still work that can be deferred.

**Comment by joewalker** [comment]
I think this is low priority. Of the potential customers in the goals doc, the only that would benefit from full text search are the more complex ones like context-stream, and sync. So we probably don't need this until H2.
